### PR TITLE
partners 2012 data set

### PR DESCRIPTION
**Pull request by davojta**

### CODE REVIEW

File: data/raw/cpm2012/partners.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  "description": "data set for cpm2012 partners",
+  "data": {
+    "partners": [
+      {
+        "name_be": "http://www.quest-europe.eu/",
+        "name_ru": "http://www.quest-europe.eu/",
+        "name_en": "http://www.quest-europe.eu/",
+        "link": "http://www.quest-europe.eu/",
+        "image": "images/qe.png",
+        "id": "qe"
```
Comment (davojta) [inline data/raw/cpm2012/partners.json]: added id for link partners across several years

File: data/raw/cpm2012/partners.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        "name_be": "http://menscity.by/",
+        "name_ru": "http://menscity.by/",
+        "name_en": "http://menscity.by/",
+        "link": "http://menscity.by/",
+        "image": "images/menscity.png",
+        "id": "menscity"
+      }
+    ],
+    "partners_list": [
+      "qe",
```
Comment (davojta) [inline data/raw/cpm2012/partners.json]: link id to not repeat partner data to same partner from other year

File: data/raw/cpm2012/parnters_list.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[
```
Comment (staaas) [inline data/raw/cpm2012/parnters_list.json]: why do we need this file?

**Comment by staaas** [comment]
LGTM

**Pull request merged**